src/components/common/YourCompany/YourCompany.vue

```diff
@@ -487,10 +484,6 @@ export default class YourCompany extends Mixins(
 
     if (this.getOriginalIA) {
       currentName = this.getOriginalIA.nameRequest.legalName
-    } else if (this.getOriginalChgRegistration) {
-      currentName = this.getOriginalChgRegistration.nameRequest.legalName
-    } else if (this.getOriginalRegistration) {
-      currentName = this.getOriginalRegistration.nameRequest.legalName
```

Comment:
Firm corrections don't load the original Registration / ChangeOfRegistration. Instead, firm corrections load a business snapshot. So, this original else case works just fine.